styles/_article.scss

```diff
@@ -86,12 +86,39 @@
     > p:last-child { margin-bottom: 0; }
   }
 
+  &-anchorbox {
+    background-color: #e8e8e8;
```

Comment:
Not critical, but we might want to create a variable for this color in a future commit since we are using it in multiple places. I just checked the www.artstor.or scss and we have these variables:
$dark-gray:      #333333;
$gray:           #e8e8e8;
$lighter-gray:   #f9f9f9;
Maybe we could add these to our variables.scss

Reply:
added those variables, replaced background-color with $gray